packages/api-client/src/main/team/member/MemberAPI.ts

```diff
@@ -85,6 +87,45 @@ class MemberAPI {
 
     return this.client.sendJSON<void>(config).then(response => response.data);
   }
+
+  public getMemberLegalHold(teamId: string, userId: string): Promise<void> {
+    const config: AxiosRequestConfig = {
+      method: 'get',
+      url: `${TeamAPI.URL.TEAMS}/${teamId}/${MemberAPI.URL.LEGAL_HOLD}/${userId}`,
+    };
+
+    return this.client.sendJSON<void>(config).then(response => response.data);
+  }
+
+  public deleteMemberLegalHold(teamId: string, userId: string): Promise<void> {
+    const config: AxiosRequestConfig = {
+      method: 'delete',
+      url: `${TeamAPI.URL.TEAMS}/${teamId}/${MemberAPI.URL.LEGAL_HOLD}/${userId}`,
+    };
+
+    return this.client.sendJSON<void>(config).then(response => response.data);
```

Comment:
```suggestion
    await this.client.sendJSON<void>(config);
```
And the same for the other functions if they don't return anything.